ucamstaffoncosts/tax.py

```diff
@@ -0,0 +1,61 @@
+import fractions
```

Comment:
consider splitting this into nic and apprenticeship modules

Reply:
I'm somewhat neutral on this. To me it make sense that all tax calculations happen in the same module but I'm not going to scream about having them split. If it's truly just "consider", then I've considered and declined. If it's "do it or I'll not approve", I'll do it.

Reply:
just consider :)

Reply:
Tested by manually plugging in different values for different schemes and comparing them to the HR tables.